### ISSUE TITLE
SAFEDIV is not included into XMILE specification

### ISSUE THREAD

**Issue by alexprey** [SDXorg/test-models#39]
Model file: https://github.com/SDXorg/test-models/blob/master/tests/xidz_zidz/xidz_zidz.xmile

This model file has wrong definition for XMILE specification http://docs.oasis-open.org/xmile/xmile/v1.0/csprd01/xmile-v1.0-csprd01.html

""SAFEDIV"" functions is not exists by specification.

**Comment by JamesPHoughton** [comment]
@wasbridge - is this one of the xmutil translated tests?

**Comment by wasbridge** [comment]
Yes -- this is a translated test from XMUtil -- XMUtil takes XIDZ and ZIDZ and turns them into a Stella specific function (missing from XMILE) called SAFEDIV(num, denom, val).  For a v2 of the spec we should probably add SAFEDIV or something like it since there is currently no clean way to do it in "native" XMILE
